package.json

```diff
@@ -2389,6 +2390,18 @@
 					"when": "(resourceScheme == pr) || (resourcePath in github:viewedFiles) || (resourcePath in github:unviewedFiles)"
 				}
 			],
+			"multiDiffEditor/resource/title": [
+				{
+					"command": "pr.markFileAsViewed",
+					"group": "navigation",
+					"when": "resourceScheme != pr && resourceScheme != review && resourceScheme != filechange && resourcePath in github:unviewedFiles"
```

Comment:
`resourcePath` isn't set here. This is an upstream issue that we need to wait for a fix for